tests/rundb/test_httpdb.py

```diff
@@ -670,88 +670,6 @@ def test_feature_vectors(create_server):
     ), "Features didn't get updated properly"
 
 
-def test_project_file_db_roundtrip(create_server):
```

Comment:
Why is this test removed?

Reply:
Moved into integration under the name `test_project_db`

Reply:
added mocking now